numpy/lib/function_base.py

```diff
@@ -1181,12 +1181,12 @@ def interp(x, xp, fp, left=None, right=None, period=None):
     One-dimensional linear interpolation.
 
     Returns the one-dimensional piecewise linear interpolant to a function
-    with given values at discrete data-points.
+    with given values at discrete data-points (`xp`, `fp`), evaluated at `x`.
```

Comment:
How about `` given values `fp` at points `xp`, evaluated at `x` ``

Reply:
@eric-wieser I like that better as well, but I sticked to the `data-points` terminology since it is used later in the parameter description for `xp` and `fp`. What do you think?

Reply:
Perhaps `passing through the discrete data-points` rather than `with given values at the discrete data-points` then. There is no value associated with the point `(xp[0], fp[0])` - the only information is the point itself.

Reply:
Right. Just updated it to simply `with given discrete data points`.